src/Athena/Athena.csproj

```diff
@@ -16,7 +16,6 @@
     <DotNetCliToolReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Tools" Version="2.0.2" />
   </ItemGroup>
   <ItemGroup>
-    <Folder Include="Content" />
```

Comment:
I **think** we need to keep this if you want the folder to show up in solution explorer but I'm not certain, we should double check this.

Reply:
the folder is still there after closing and re-opening VS